clifford/_multivector.py

```diff
@@ -970,12 +921,12 @@ def join(self, other) -> 'MultiVector':
             raise ValueError("not blades")
 
     def meet(self, other, subspace=None) -> 'MultiVector':
-        r"""Returns the meet of two blades.
+        r"""The meet of two blades.
 
         Computation is done with respect to a subspace that defaults to
         the join if none is given.
-                     -1
-        M \/i N = (Mi  ) * N
+
+        :math:`M \vee_i N = M_i^{-1}N`
```

Comment:
I have no idea what this means, but is my best guess at the intent of the ascii art